authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -45,8 +45,10 @@ metadata:
   namespace: <target-namespace> <1>
   name: <target-secret-name> <2>
 data:
```

Comment:
@jeana-redhat I read @joelddiaz 's review comment, the `data` would be `stringData` .
data field will show a base64 encode string while stringData shows a readable string.

Reply:
Thanks, I apparently could not read. Will fix that and get it updated :+1: